src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,186 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    public static void main(String[] args) {
+
+        // Welcome Message
+        printWelcomeMessage();
+
+        // Active Chat
+        activeChat();
+
+        // Goodbye Message
+        printGoodbyeMessage();
+    }
+
+    private static void activeChat() {
+        boolean isBye = false;
+        String input;
+        Scanner in = new Scanner(System.in);
+        int counter = 0;
+        Task[] list = new Task[100];
+
+        while(!isBye){
+            //store input
+            input = in.nextLine();
+            //process input
+            if (input.equals("bye")){ //check if bye
+                isBye = true;
+            } else if (input.equals("list")) { //check if list
+                processList(counter, list);
+            } else if (input.contains("done") ) { //check if done
+                processDone(input, list);
+            } else {
+                try {
+                    counter = processTasks(input, counter, list); //process tasks
+                } catch ( IllegalTaskException e ) {
+                    System.out.println("Please include /by for deadline and /at for event");
+                }
+            }
+        }
+    }
```

Comment:
Consider implementing a Parser class to parse user inputs instead of putting everything in main/Duke